src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -19,14 +19,37 @@ void EnRz_Draw(Actor* thisx, PlayState* play);
 void EnRz_ActorShadowFunc(Actor* thisx, Lights* mapper, PlayState* play);
 void EnRz_ChangeAnim(PlayState*, EnRz*, s16 animIndex, u8 animMode, f32 transitionRate);
 s32 func_80BFBA50(EnRz* this, PlayState* play);
-EnRz* func_80BFBDA0(EnRz* this, PlayState*);
+EnRz* EnRz_FindSister(EnRz* this, PlayState*);
 void func_80BFC058(EnRz* this, PlayState* play);
 void func_80BFC078(EnRz* this, PlayState* play);
 void func_80BFC3F8(EnRz* this, PlayState* play);
 void func_80BFC674(EnRz* this, PlayState* play);
 void func_80BFC7E0(EnRz* this, PlayState* play);
 void func_80BFC8F8(EnRz* this, PlayState* play);
 
+typedef enum {
+    /* 0 */ EN_RZ_ROSA_SISTERS_JUDO,
+    /* 1 */ EN_RZ_ROSA_SISTERS_MARILLA
+} EnRzRosaSisters;
+
+typedef enum {
+    /* 1 */ EN_RZ_TYPE_1 = 1,
+    /* 2 */ EN_RZ_TYPE_2
+} EnRzType;
```

Comment:
These would be used at least by other files (like rooms, eventually), so should go in the header as usual. There is also a type 0:
```
    { ACTOR_EN_RZ,
      { 41, 148, -113 },
      { SPAWN_ROT_FLAGS(0, 0x0001), SPAWN_ROT_FLAGS(0X5A, 0x007F), SPAWN_ROT_FLAGS(0, 0x0020) },
      0x0A00 },
```

so that case needs to be part of `EnRzType`. And the first one should be
```c
typedef enum {
    /* 0 */ EN_RZ_SISTER_JUDO, // in red
    /* 1 */ EN_RZ_SISTER_MARILLA // in blue
} EnRzRosaSister;
```

imo, the whole actor is the Rosa sisters, which you could use instead of EnRz/EN_RZ in the enum names, this enum is just which it is.

I'd also comment on the colours since I doubt most people know their actual names.